src/pages/SettingsPage.js

```diff
@@ -86,6 +113,25 @@ const SettingsPage = ({
                         {Str.removeSMSDomain(session.email)}
                     </Text>
                     )}
+                    <View style={[styles.settingsPageBody, styles.mt4]}>
+                        <Text style={[styles.textLabel, styles.textStrong, styles.mt2, styles.mb2]} numberOfLines={1}>
+                            Priority Mode
+                        </Text>
+                        <View style={[styles.mt2, styles.mb2]}>
```

Comment:
Can we get rid of the `mt2` here?

Reply:
![image](https://user-images.githubusercontent.com/47184118/109482597-1ad79e00-7a76-11eb-9961-4efe20eb68ae.png)

Reply:
This looks better, thanks!